recipes/tagmaps/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "tagmaps" %}
+{% set version = "0.17.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 750b67b3daffcdcda3302910437a1d70150a2a60e53f63665ea8d5d3d4805ccd
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - tagmaps = tagmaps.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python>=3.5
```

There needs to be a space before the version constraint else creating the feedstock will fail:
```suggestion
    - python >=3.5
```
This also applies to the run dependencies.

---

Great, thanks for the commit!